dev/Kozani/KozaniManagerRuntime/M.K.MR.ManagerRuntimeManager.cpp

```diff
@@ -9,6 +9,16 @@
 
 namespace winrt::Microsoft::Kozani::ManagerRuntime::implementation
 {
+    ManagerRuntimeManager::ManagerRuntimeManager()
+    {
+        m_kozaniManager = wil::CoCreateInstance<KozaniManager, IKozaniManager>(CLSCTX_LOCAL_SERVER);
```

Comment:
Seems a bit odd to do in the ctor but the ctor's private, right? Only way to get here is via the static Create()?

Reply:
we cannot put ctor in private as it will cause cpp/winrt build error. I tried and it failed. It is the same pattern as all the other APIs in this solution. None of them have private ctor.